src/content/api/bank-account-connection-intents.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Bank Account Connection Intents
+description: Bank Account Connection Intent model and related endpoints
+draft: true
+nav:
+  path: API
+  order: 18
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import CodeGroup from '../../components/CodeGroup.astro';
+import Badge from '../../components/Badge.astro';
+
+A Bank Account Connection Intent facilitates user authorization of access to Bank Accounts, for example, using Open Banking flows. Typically the authorization will follow an OAuth flow where the user is redirected to a third-party system to authenticate and select one or more bank accounts.
+
+## Bank Account Connection Intent Model
```

Comment:
Suggestion: Just call this model to simplify the navigation
<img width="302" alt="image" src="https://github.com/centrapay/centrapay-docs/assets/64108933/44391c6d-1bf4-4a86-9afa-02b08c48b46e">